internal/runners/packages/uninstall.go

```diff
@@ -12,7 +12,7 @@ import (
 
 // UninstallRunParams tracks the info required for running Uninstall.
 type UninstallRunParams struct {
-	Package captain.PackageValueNoVersion
+	Packages captain.PackagesValue
```

Comment:
Was this an intentional change? Now users can use `state uninstall foo@x.y`, right?

Reply:
Ah right, didn't realize uninstall has a different version. I'll add a new value type for it.